content/ADR-117-CRDT-protocol.md

```diff
@@ -41,11 +41,19 @@ That problem magnifies the "queue optimization" problem of the interpreter of me
 
 ### Impossible use cases
 
-> TODO: Static scenes
-
-> TODO: Synchronization of state over network
-
-> TODO: Load parcels at a distance, then load the code. Shut down the code when moving away
+## Static Scenes & Load Parcels
+Besides the limitation of the messages being sent, another limitation that we are facing was how the static scenes were implemented, and how bad they performed.
+No matter what kind of scene you have, for this scene to be able to work in Decentraland we need to download the game.js and eval that code to run the scene. Even if its only a static house with some trees, or even an empty road.
+With the new approach of CRDT's, we can deploy a file containing the relation between the entities and components (CRDT State), and download only that file with the models, instead of downloading the game.js and doing an eval of the code.
```

Comment:
```suggestion
With the new approach of CRDTs, we can deploy a file containing the relation between the entities and components (CRDT State) and download only that file with the models instead of downloading the game.js and doing an eval of the code.
```